content/200-concepts/300-more/400-comparisons/04-prisma-and-drizzle.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Drizzle'
+metaTitle: 'Prisma vs Drizzle'
+metaDescription: 'Learn how Prisma compares to Drizzle.'
```

Comment:
I think they call themselves "Drizzle ORM" consistently. We should decide if we want to honor that, or make a point here. (I think I would prefer leaving it off)